Lib/test/test_sys_settrace.py

```diff
@@ -2064,6 +2064,40 @@ def test_jump_simple_backwards(output):
         output.append(1)
         output.append(2)
 
+    @jump_test(1, 4, [2])
+    def test_jump_is_none_forwards(output):
+        x = None
+        if x is None:
+            output.append(1)
+        else:
+            output.append(2)
```

Comment:
Not a hard rule, but the general style in these tests is to append the *line number* to `output`, so it's a bit easier to keep track of where we've been. Example:

```suggestion
    @jump_test(1, 4, [5])
    def test_jump_is_none_forwards(output):
        x = None
        if x is None:
            output.append(3)
        else:
            output.append(5)
```

Reply:
Ah that makes sense - a little easier to reason about.